include/seastar/core/chunked_fifo.hh

```diff
@@ -296,6 +298,25 @@ chunked_fifo<T, items_per_chunk>::chunked_fifo(chunked_fifo&& x) noexcept
     x._nfree_chunks = 0;
 }
 
+template <typename T, size_t items_per_chunk>
+inline
+chunked_fifo<T, items_per_chunk>::chunked_fifo(const chunked_fifo& rhs)
+        : chunked_fifo() {
+    std::copy_n(rhs.begin(), rhs.size(), std::back_inserter(*this));
```

Comment:
This is suboptimal (better to use uninitialized_copy_n so it can translate to memcpy), but not worth optimizing now.